src/reducers/xdrViewer.js

```diff
@@ -37,3 +37,15 @@ function type(state = 'TransactionEnvelope', action) {
 
   return state;
 }
+
+function fetchedSigners(state = "uninitialized", action) {
```

Comment:
We can use `undefined` here (or even remove value as it will be `undefined` by default?). Or maybe add:
```js
export const FETCHED_SIGNERS_PENDING = 'FETCHED_SIGNERS_PENDING'; // or a better name?
```

Reply:
Originally that was undefined, specifically an empty string, however in js land ```"" == []``` which failed for this:

```
// Fetch signers on initial load
  if (state.type === "TransactionEnvelope" && state.fetchedSigners === 'uninitialized') {
    dispatch(fetchSigners(state.input, baseURL, networkPassphrase))
  }
```

since initial state == case with no signers ([])

Reply:
Also, default state cannot be undefined:
```
combineReducers.js:56 Uncaught Error: Reducer "fetchedSigners" returned undefined during initialization. If the state passed to the reducer is undefined, you must explicitly return the initial state. The initial state may not be undefined. If you don't want to set a value for this reducer, you can use null instead of undefined.
```

Reply:
Decided to go with:

INITIAL -->     null
SUCCESS --> result (type array)
FAILURE -->      "ERROR"
PENDING -->     "PENDING"